apps/console-lite/src/app/components/deal-ticket/deal-ticket-steps.tsx

```diff
@@ -179,7 +179,7 @@ export const DealTicketSteps = ({
           ItemRenderer={MarketNameRenderer}
```

Comment:
can't see related changes in mocks in unit tests and console-lite-e2e mock generators

Reply:
Done - perhaps it'd be good to consider coupling the mocks with generated types